### DISCUSSION TITLE
Suricata is missing so-status

### DISCUSSION THREAD

**Discussion by techn00** [Security-Onion-Solutions/securityonion#7324] [Unsupported Versions] [answered]
I have installed Security onion using latest ISO image on Hyper V hypervisor with 500GB disk, 32GB Ram, 8 vCPU
Post installation, I checked so-status, it shows suricata is missing. Please any help to fix this issue.

![image](https://user-images.githubusercontent.com/100316438/155709574-73710fba-baeb-42f8-8f94-f231f4e65540.png)

**Comment by Acewiza** [comment]
I would probably just re-run the install, then dissect those logs if it happens again.

**Comment by techn00** [comment, marked as answer]
I have re-run the setup but I am getting this error as soon as the installation start, here is the screenshot from sosetup.log file

![image](https://user-images.githubusercontent.com/100316438/155735543-80f7a074-6a25-4cc5-8a38-2fd308c60523.png)

**Reply by Acewiza** [reply]
That seems to suggest it's having network issues.

**Reply by techn00** [reply]
Thanks for your help. Issue was resolve after fixing network issues and re-run the installation.
